### PR TITLE
fixed navbar with flexbox

### PR DESCRIPTION
**Pull request by Dcamo001**

### CODE REVIEW

**Comment by Dcamo001** [comment]
Still need to:
- Remove underline and blue hyperlink color from all items
- Keep Title black at all times, and "current page" black, the rest should be gray

**Comment by awitherow** [comment]
removing underline is easy "text-decoration" and then hyperlink properties can have "visited", "hover" on the a element.

as said for the current page you will need to do some javascript on that most likely. will be fun to figure that out :)

File: about Fern and Bee.html [hunk 1]
```diff
@@ -11,36 +11,21 @@
...
-    <ul>
-      <li><a href="./portfolio.html">portfolio</a></li>
-      <li><a href="./blog.html">blog</a></li>
-      <li><a href="./about Fern and Bee.html">about Fern and Bee</a></li>
-      <li><a href="./purchase.html">purchase</a></li>
-      <li><a href="./contact.html">contact</a></li>
+  <!-- navbar -->
+  
+    <ul class="navbar" role="navigation">
+      <h1 class="main-title"><a href="./index.html"> FERN AND BEE </a></h1>
```
Comment (awitherow) [inline about Fern and Bee.html]: it is typically not a good idea to nest anything BUT list items in an `ul` or `ol`.

I would make it more like...

```
<div class="navbar">
  <h1 />
  <ul />
</div>
```

and use `.navbar` to do wrapper styling. `.navbar h1` and `.navbar ul` to access the specific h1, ul styling.

File: about Fern and Bee.html [hunk 2]
```diff
@@ -11,36 +11,21 @@
...
+      <li class="navbar-items"><a href="./portfolio.html">portfolio</a></li>
```
Comment (awitherow) [inline about Fern and Bee.html]: no need to do `.navbar-items` rather `.navbar ul li` without needing to do a class.

File: about Fern and Bee.html [hunk 3]
```diff
@@ -11,36 +11,21 @@
...
+      <li class="navbar-items"><a href="./blog.html">blog</a></li>
+      <li class="navbar-items"><a href="./about Fern and Bee.html">about Fern and Bee</a></li>
```
Comment (awitherow) [inline about Fern and Bee.html]: just for aesthetics I would change this to only `about.html`

File: styles/style.css [hunk 4]
```diff
@@ -1,176 +1,143 @@
 html {
```
Comment (awitherow) [inline styles/style.css:1]: i am not sure what a lot of these styles are used for on the html.

top, right -> use margins instead

width, height -> not sure why you are constraining width and height for entire html element.

font-size is defined twice.

background-color shouldn't need !important?

also the line-height, letter spacing might be better off on more specific elements like p, h1, etc... or you might get unwanted side effects later.

File: styles/style.css [hunk 5]
```diff
@@ -1,176 +1,143 @@
...
-
-ul {
-  list-style-type: none;
-  margin: 0;
-  padding: 0;
-  overflow: hidden;
   background-color:white;
+  width:             1100px;
+  margin:            0 auto;
+  padding:           0px 20px 20px 20px;
```
Comment (awitherow) [inline styles/style.css]: any time you reference a 0, you do not need the px for padding and margin.

File: styles/style.css [hunk 6]
```diff
@@ -1,176 +1,143 @@
...
-  text-decoration: none;
-}
-
-li a:hover {
-  color: #111;
-}
+a:active {
+  text-decoration:   none;
+  font-weight:       bold;
+  color:           black!important;
```
Comment (awitherow) [inline styles/style.css:17]: just being picky and saying check out the indentation here ;)

File: styles/style.css [hunk 7]
```diff
@@ -1,176 +1,143 @@
...
-  transform: scale(1, 1.05);
+  font-size:         13px;
+  text-align:        justify;
+  font-weight:       300;
+  color:           gray;
+  justify-content:   space-between;
+  letter-spacing:    1.15px;
+  line-height:       .9;
+  margin-bottom:     -15px;
+  transform:         scale(1, 1.05);
```
Comment (awitherow) [inline styles/style.css]: just a hint this will effect ALL divs, might be more wise to put some of these in a specific class.

**Pull request merged**